src/r2mm/launching/instructions/instructions/loader/SkulTheHeroSlayerGameInstructions.ts

```diff
@@ -0,0 +1,60 @@
+import GameInstructionGenerator from '../GameInstructionGenerator';
+import { GameInstruction } from '../../GameInstructions';
+import Game from '../../../../../model/game/Game';
+import Profile from '../../../../../model/Profile';
+import FsProvider from '../../../../../providers/generic/file/FsProvider';
+import path from 'path';
+import { DynamicGameInstruction } from '../../DynamicGameInstruction';
+import { GameInstanceType } from '../../../../../model/game/GameInstanceType';
+
+export default class SkulTheHeroSlayerGameInstructions extends GameInstructionGenerator {
```

Comment:
Any way we can avoid creating an entirely game-specific dedicated instruction class just for skul, and instead extend the generic instructions?